contracts/TokenConverter/AbstractTokenConverter.sol

```diff
@@ -686,6 +739,79 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
         conversionPaused = false;
     }
 
+    /// @dev _updateAssetsState hook to update the states of reserves transferred for the specific comptroller
+    /// @param comptroller Comptroller address (pool)
+    /// @param asset Asset address
+    function _updateAssetsState(address comptroller, address asset) internal virtual returns (uint256) {}
+
+    /// @dev This method is used to convert asset into base asset by converting them with other converters which supports the pair and transfer the funds to
+    /// destination contract as destination's base asset
+    /// @param comptroller Comptroller address (pool)
+    /// @param tokenAddressOut Address of the token transferred to converter, and through _privateConversion it will be converted into base asset
+    /// @param balanceDiff Amount of the tokenAddressOut transferred to converter
+    function _privateConversion(
+        address comptroller,
+        address tokenAddressOut,
+        uint256 balanceDiff
+    ) internal {
+        address tokenAddressIn = _getDestinationBaseAsset();
+        (address[] memory converterAddresses, uint256[] memory converterBalances) = IConverterNetwork(converterNetwork)
```

Comment:
I think we should check first if `conversionConfigurations[tokenAddressOut][tokenAddressIn].enabled` is true on this contract

Reply:
And maybe we should add an extra field to enable/disable private conversions per token (in the origin, in the TokenConverter contract that initiates the private conversion). This way we could disable private conversions for fee-on-transfer tokens. Wdyt?

Reply:
https://github.com/VenusProtocol/protocol-reserve/pull/35#issuecomment-1792266120